programs/programs/nc/medicaid/nc_medicare_savings/calculator.py

```diff
@@ -0,0 +1,65 @@
+from programs.programs.calc import MemberEligibility
+from programs.programs.federal.medicare_savings.calculator import MedicareSavings
+from typing import ClassVar
+
+
+class MedicareSavingsNC(MedicareSavings):
+    ineligible_insurance_types: ClassVar[tuple[str, ...]] = ("va", "medicaid")
+    asset_limit: ClassVar[dict[str, int]] = {"single": 9_660, "married": 14_470}
+    min_income_percent: ClassVar[float] = 1.0
+
+    dependencies: ClassVar[list[str]] = [
+        "household_assets",
+        "relationship",
+        "income_frequency",
+        "income_amount",
+        "age",
+        "insurance",
+    ]
+
+    def member_eligible(self, e: MemberEligibility):
+        super().member_eligible(e)
```

Comment:
I see you have extended the MedicareSavings class, great! I think there still is opportunity here to combine these more; I see a lot of duplicate logic in here that may not make sense because you are already calling `super().member_eligible(e)` which should already check a lot of the conditions. I think it would be helpful to understand, how is the logic for MedicareSavings different in NC than we have already in the federal logic? Then extend the class to support it. 

For example, the main difference I see is the min_income check. Is that NC specific, or true to federal as well? 

cc @Lkp2104